install/kubernetes/cilium/values.yaml

```diff
@@ -1680,6 +1705,17 @@ clustermesh:
     # -- Labels to be added to clustermesh-apiserver pods
     podLabels: {}
 
+    # -- PodDisruptionBudget settings
+    # ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
+    podDisruptionBudget:
+      # -- enable PodDisruptionBudget
+      enabled: false
+      # -- Minimum number/percentage of pods that should remain scheduled.
+      # When it's set, maxUnavailable must be disabled by `maxUnavailable: null`
+      minAvailable: null
+      # -- Maximum number/percentage of pods that may be made unavailable
+      maxUnavailable: 1
```

Comment:
@lic17 General look good, but one last thing:
![image](https://user-images.githubusercontent.com/6848311/148188804-dd39a91b-a864-4c96-87e1-7ebb2eeb54a8.png)
Because, `enabled`, `minAvailable` and `maxUnavailable` are documented, so I think `podDisruptionBudget` object is no need to documented. You can do this by remove this `--`
```suggestion
    # PodDisruptionBudget settings
    # ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
    podDisruptionBudget:
      # -- enable PodDisruptionBudget
      enabled: false
      # -- Minimum number/percentage of pods that should remain scheduled.
      # When it's set, maxUnavailable must be disabled by `maxUnavailable: null`
      minAvailable: null
      # -- Maximum number/percentage of pods that may be made unavailable
      maxUnavailable: 1
```

Reply:
have done